src/logic/contracts/safeContracts.ts

```diff
@@ -19,7 +20,7 @@ export const SAFE_MASTER_COPY_ADDRESS_V10 = '0xb6029EA3B2c51D09a50B53CA8012FeEB0
 let proxyFactoryMaster
 let safeMaster
 
-const createGnosisSafeContract = (web3) => {
+const createGnosisSafeContract = (web3: Web3): any => {
```

Comment:
Is possible to return `unknown` instead of `any`?

Reply:
Using any for contracts is OK for now, because with unknown you will have to do something like this

if (contract)
if (contract.methods)
if (contract.methods.etc)

I'm working on adding [typechain](https://github.com/ethereum-ts/TypeChain) to type smart contracts, so this will be addressed soon

Reply:
done and reverted :grimacing: